sys/net/gnrc/network_layer/ipv6/nib/nib_ft.c

```diff
@@ -45,12 +46,20 @@ int gnrc_ipv6_nib_ft_add(const ipv6_addr_t *dst, unsigned dst_len,
     }
     mutex_lock(&_nib_mutex);
     if (is_default_route) {
-        ptr = _nib_drl_add(next_hop, iface);
+        _nib_dr_entry_t *dr = _nib_drl_add(next_hop, iface);
+        if (dr) {
+            dr->ltime = ltime;
+            ptr = (void *) dr;
+        }
     }
 #if GNRC_IPV6_NIB_CONF_ROUTER
     else {
         dst_len = (dst_len > 128) ? 128 : dst_len;
-        ptr = _nib_ft_add(next_hop, iface, dst, dst_len);
+        _nib_offl_entry_t *offl = _nib_ft_add(next_hop, iface, dst, dst_len);
+        if (offl) {
+            offl->valid_until = offl->pref_until = ltime * MS_PER_SEC;
```

Comment:
Iff we add this feature we need to add a new field to `_nib_offl_entry_t`. You are risking here to override prefix lifetimes and thus causing potential side effects.